### PR TITLE
Branding changes

### PR DESCRIPTION
**Pull request by kaushalvivek**
Changes made:
- MLFlow Homepage logo changed to Couture logo
- MLFlow wheel changed to Couture icon
- favicon updated
- GitHub link changed to coutureai/mlflow

### CODE REVIEW

File: mlflow/server/js/src/experiment-tracking/components/App.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ class App extends Component {
                 </NavLink>
               </div>
               <div className='header-links'>
-                <a href={'https://github.com/mlflow/mlflow'}>
+                <a href={'https://github.com/coutureai/mlflow'}>
```
Comment (Anubhav2701) [inline mlflow/server/js/src/experiment-tracking/components/App.js]: Maybe just keep the url as https://couture.ai

**Review by Anubhav2701 — APPROVED** [review]

File: mlflow/server/js/src/experiment-tracking/components/App.js (hunk 1 above)
Comment (kaushalvivek) [reply mlflow/server/js/src/experiment-tracking/components/App.js]: The link for the text is 'GitHub', should I change it to "Couture.ai" or something else?

Comment (Anubhav2701) [reply mlflow/server/js/src/experiment-tracking/components/App.js]: change it to couture.ai as well

Comment (kaushalvivek) [reply mlflow/server/js/src/experiment-tracking/components/App.js]: Sure

Comment (kaushalvivek) [reply mlflow/server/js/src/experiment-tracking/components/App.js]: Corrected in fe4c4f7

**Pull request merged**